mgmtsystem/rest_client.py

```diff
@@ -57,10 +57,11 @@ def get(self, entity_type, name=None, namespace=None):
         r = self.raw_get(path)
         return (r.status_code, r.json() if r.ok else None)
 
-    def get_json(self, path):
-        r = self.raw_get(path)
+    def get_json(self, path, headers={"Hawkular-Tenant": "hawkular"}):
```

Comment:
Looking at this, I think other mgmt_systems are using this rest_client and the get_json call, so I think that headers should be None by default and you pass in your tenant in your calls, maybe create a partial in your module.